config/pom.xml

```diff
@@ -147,6 +147,34 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
+
+        <dependency>
```

Comment:
From what I cant see you are not actually using any of these dependencies?

Reply:
com.diffblue.deeptestutils is used explicitly, the rest are used implicitly. Two can be removed now though as I removed the tests relying on mocking.